torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10666,18 +10672,32 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     OpInfo('nn.functional.conv_transpose2d',
            aten_name='conv_transpose2d',
            aliases=('conv_transpose2d',),
-           dtypes=floating_types_and(torch.int64),
-           dtypesIfCUDA=floating_types_and(torch.float16,
-                                           *[torch.bfloat16] if (CUDA11OrLater or TEST_WITH_ROCM) else []),
+           # `ref` for this function is backward of
+           # corresponding `conv*d`
+           ref=partial(conv_transpose_ref, fn=torch.nn.functional.conv_transpose2d),
+           dtypes=floating_and_complex_types_and(torch.int64),
+           dtypesIfCUDA=floating_and_complex_types_and(torch.float16, torch.chalf,
+                                                       *[torch.bfloat16] if (CUDA11OrLater or TEST_WITH_ROCM) else []),
            sample_inputs_func=sample_inputs_conv_transpose2d,
+           # Runs very slowly on slow-gradcheck for complex.
+           gradcheck_fast_mode=True,
```

Comment:
Even with decreased sizes it takes a lot of time.

With slow-gradcheck, it takes more than 25 mins (had to interrupt the tests)
```
=================================================== 12 passed, 10 skipped, 15264 deselected in 1542.03s (0:25:42) ================================================
```

With fast-mode (17 secs),
```
========================================================= 16 passed, 16 skipped, 15264 deselected in 17.72s =======================================================
```

cc: @soulitzer

Reply:
shouldn't we just always run fast gradcheck by default? IIRC slow gradcheck is only useful for debugging, right?

Reply:
This sounds good.

@anjali411 we still run periodic tests with slow gradcheck as default, so explicitly setting fast_mode=True would help prevent timing out there